buddyboss/buddyboss-platform
# Reflected XSS when describing an image + other security fixes

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1568 

### How to test the changes in this Pull Request:

1. Go to an image which is part of a message (for example a message to someone, or a post in the forum)
2. Click on the image (to make the description/replies to it visible)
3. Click on 'add a description' or 'Edit' in the description area of the image.
4. Add a piece of javascript, for example '<script src=http://xss.rocks/xss.js? >'
5. See the XSS alert gone.

### Other information:
